prisma/schema.prisma

```diff
@@ -0,0 +1,105 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+// Looking for ways to speed up your queries, or scale easily with your serverless or edge functions?
+// Try Prisma Accelerate: https://pris.ly/cli/accelerate-init
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+model GovernanceAuthority {
+  id               String                @id @default(uuid())
+  name             String
+  did              String                @unique
+  organizations    Organization[]
+  namespaces       Namespace[]
+  assuranceLevels  AssuranceLevel[]
+  ownedRelationships    RegistryRelationship[] @relation("OwningRegistry")
+  relatedToRelationships RegistryRelationship[] @relation("RelatedRegistry")
+}
+
+model Organization {
+  id                    String            @id @default(uuid())
+  name                  String
+  did                   String            @unique
+  type                  OrganizationType
+  governanceAuthorityId String
+  governanceAuthority   GovernanceAuthority @relation(fields: [governanceAuthorityId], references: [id])
+  onboardedAt           DateTime          @default(now())
+  schemas               Schema[]
+}
+
+model Schema {
+  id              String       @id @default(uuid())
+  name            String
+  definition      Json
+  organizationId  String
+  organization    Organization @relation(fields: [organizationId], references: [id])
+  credentials     Credential[]
+}
+
+model Credential {
```

Comment:
This can be removed, as credential details are not stored anywhere

Reply:
All the reference to other table should also be removed

Reply:
Removed the Credential model and all its references from the schema.